code/game/objects/effects/spawners/f13lootdrop.dm

```diff
@@ -383,15 +383,47 @@
 				/obj/effect/spawner/bundle/f13/armor/combat/mk2/dark
 				)
 
-/obj/effect/spawner/lootdrop/f13/armor/tier4PA // TIER 4 ARMOR WITH PA
-	name = "tier 4 armor with possible PA"
+/obj/effect/spawner/lootdrop/f13/armor/tier4PA // Not actually tier4 but too lazy to rename/repath and remap. Use this very sparingly as it has PA.
+	name = "tier between 4 and 5 armor with possible PA"
 	loot = list(
-				/obj/effect/spawner/bundle/f13/armor/t45d,
-				/obj/effect/spawner/bundle/f13/armor/riot,
-				/obj/effect/spawner/bundle/f13/armor/combat/mk2,
-				/obj/effect/spawner/bundle/f13/armor/combat/mk2/dark
+				/obj/effect/spawner/bundle/f13/armor/advanced = 4,
+				/obj/effect/spawner/bundle/f13/armor/t51b = 6,
+				/obj/effect/spawner/bundle/f13/armor/t45d = 8,
+				/obj/effect/spawner/bundle/f13/armor/t45b = 10,
+				/obj/effect/spawner/bundle/f13/armor/eliteriot = 20,
+				/obj/effect/spawner/bundle/f13/armor/desertriot = 20,
+				/obj/effect/spawner/bundle/f13/armor/policeriot = 20,
```

Comment:
The armor for this spawner doesn't have a sprite.

Reply:
Fixed.